slimmer/slimmer.cpp

```diff
@@ -160,14 +168,14 @@ int parsed_main(int argc, char** argv) {
                       << reliso << "]" << std::endl;
           }
           em_directions.emplace_back(lep.eta(), lep.phi());
+          output.n_lep_loose++;
           if (is_loose()) {
-            lepvec += lep.p4();
-            output.n_lep_loose++;
```

Comment:
@DylanHsu Here, I moved from counting loose leptons that passed the loose cut to counting leptons that pass the preselection. I got the impression from the AN that the preselection is used for cleaning, but the loose counting is done with the MVA/muon working points. I could be wrong about that though.

Reply:
for what it's worth I think that is the right interpretation